app/ui/legacy/src/main/res/layout/text_divider_list_item.xml

```diff
@@ -13,6 +13,5 @@
     android:paddingRight="16dp"
     android:paddingTop="4dp"
     android:singleLine="true"
-    android:textAppearance="?attr/textAppearanceSubtitle2"
-    android:textColor="?attr/colorAccent"
```

Comment:
We should probably keep `android:textColor="?attr/colorSecondary"`.

This layout is used for section headers in the settings screen. The text there was intentionally colored using the accent/secondary color. It matches the About screen and what AndroidX's `PreferenceCategory` is doing.